theme.json

```diff
@@ -28914,6 +28914,13232 @@
+                    }
+                  }
+                ],
+                "breakpoints": {
+                  "tablet": {
+                    "name": "smallCardListItemShadowSpreadTablet",
+                    "default": "inherit-mobile"
+                  },
+                  "desktop": {
+                    "name": "smallCardListItemShadowSpreadDesktop",
+                    "default": "inherit-tablet"
+                  }
+                },
+                "label": "Spread",
+                "type": "size",
+                "subtype": "shadow"
+              },
+              {
+                "name": "smallCardListItemShadowColor",
+                "default": "rgba(51, 51, 51, 0.2)",
+                "styles": [
+                  {
+                    "parentSelector": "[data-widget-package='com.fliplet.dynamic-lists'] [data-dynamic-lists-layout='small-card']",
+                    "selectors": [".new-small-card-list-container .small-card-list-wrapper .small-card-list-item"],
+                    "properties": ["box-shadow"],
+                    "type": "shadow",
+                    "siblings": {
+                      "style": "smallCardListItemShadowStyle",
+                      "xoffset": "smallCardListItemShadowXoffset",
+                      "yoffset": "smallCardListItemShadowYoffset",
+                      "blur": "smallCardListItemShadowBlur",
+                      "spread": "smallCardListItemShadowSpread",
+                      "color": "smallCardListItemShadowColor"
+                    }
+                  }
+                ],
+                "breakpoints": {
+                  "tablet": {
+                    "name": "smallCardListItemShadowColorTablet",
+                    "default": "inherit-mobile"
+                  },
+                  "desktop": {
+                    "name": "smallCardListItemShadowColorDesktop",
+                    "default": "inherit-tablet"
+                  }
+                },
+                "type": "color"
+              }
+            ]
+          },
+          {
+            "description": "Visibility",
+            "appSupportsContainers": true,
+            "fields": [
+              {
+                "name": "smallCardVisibility",
+                "default": "visible",
+                "columns": 12,
+                "breakpoints": {
+                  "tablet": {
+                    "name": "smallCardVisibilityTablet",
+                    "default": "inherit-mobile"
+                  },
+                  "desktop": {
+                    "name": "smallCardVisibilityDesktop",
+                    "default": "inherit-tablet"
+                  }
+                },
+                "type": "toggle",
+                "subtype": "visibility"
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "name": "Feed - List (from data source)",
+        "packages": ["com.fliplet.dynamic-lists%3Anews-feed"],
```

Comment:
Same here, I though this would be decoded. Can we use `:` instead of `%3A` will make it easier for other developers to understand.

Reply:
Fixed